cypress/integration/createBoxesAndProducts.js

```diff
@@ -0,0 +1,106 @@
+import uuidv4 from "uuid/v4";
+
+describe('Create and delete boxes', function() {
+    let testUserMail;
+    let testPwd;
+
+    before(function() {
+        cy.getTestData().then(($result) => {
+            testUserMail = $result.testUserMail;
+            testPwd = $result.testPwd;
+        });
+    })
+
+    beforeEach(function() {        
+        cy.reLogin(testUserMail, testPwd);
+    });
+
+    afterEach(function() {
+        cy.wait(2000);
```

Comment:
How come?

Reply:
It was just for debugging purposes. Forgotten to delete. Fixed.